### PR TITLE
feat(asciidoc) enable the include directive

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

File: packages/@honkit/asciidoc/src/toHTML.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ const asciidoctor = Asciidoctor();
 
 // Render Asciidoc to HTML (block)
 function asciidocToHTML(content: string) {
-    return asciidoctor.convert(content, { attributes: "showtitle" });
+    return asciidoctor.convert(content, { safe: "server", attributes: "showtitle" });
```
Comment (azu) [inline packages/@honkit/asciidoc/src/toHTML.ts:7]: https://asciidoctor.org/docs/user-manual/#running-asciidoctor-securely
Does this PR change UNSAFE → SERVER?

**Review by azu — APPROVED** [review]
LGTM

File: packages/@honkit/asciidoc/src/toHTML.ts (hunk 1 above)
Comment (Mogztter) [reply packages/@honkit/asciidoc/src/toHTML.ts:7]: No the default value when using the API is `SECURE`

Comment (azu) [reply packages/@honkit/asciidoc/src/toHTML.ts:7]: I see. Thanks

**Pull request merged**